runtime/commands/default.js

```diff
@@ -740,6 +740,147 @@ Commands.ban = {
   }
 }
 
+Commands.hackban = {
+  name: 'hackban',
+  help: 'Swing the ban hammer on someone who isn\'t a member of the server!',
+  noDM: true,
+  usage: '<userid | userids> <optional reason>',
+  level: 0,
+  fn: function (msg, suffix, bot) {
+    var guildPerms = msg.author.permissionsFor(msg.guild)
```

Comment:
Your variable declarations are inconsistent at this point... Here you use `var`, but at L801 you use `let`. I suggest you use `let` consistently to avoid issues with JS variable hoisting.